eventmesh-protocol-plugin/eventmesh-protocol-cloudevents/src/main/java/org/apache/eventmesh/protocol/cloudevents/resolver/http/SendMessageRequestProtocolResolver.java

```diff
@@ -62,6 +62,10 @@ public static CloudEvent buildEvent(Header header, Body body) throws ProtocolHan
 
             CloudEvent event = null;
             if (StringUtils.equals(SpecVersion.V1.toString(), protocolVersion)) {
+                if(EventFormatProvider.getInstance() == null ||
```

Comment:
please use `Object.requireNonNull` instead of return `null`

Reply:
ok.,done. please review again